### PR TITLE
yuicompressor: add openjdk as dependency

### PR DESCRIPTION
**Pull request by sarisia**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

`yuicompressor` requires Java [as described here](https://yui.github.io/yuicompressor/),
but current Formulae does not specify `openjdk` as dependency.

Also, current Formulae makes `brew test yuicompressor` fail.

This PR adds `openjdk` as a dependency of `yuicompressor`.

### CODE REVIEW

File: Formula/yuicompressor.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@ class Yuicompressor < Formula
```
Comment (SMillerDev) [inline Formula/yuicompressor.rb:6]:
```suggestion
revision 1
license "BSD-3-Clause"

```

**Review by SMillerDev — COMMENTED** [review]
Adding these will update it for everyone (and indicate the license)

File: Formula/yuicompressor.rb (hunk 1 above)
Comment (sarisia) [reply Formula/yuicompressor.rb:8]: updated, thank you!

**Comment by jonchang** [comment]
Thanks for contributing to Homebrew @sarisia! This has been merged.

**Pull request merged**